scripts/globals/battlefield.lua

```diff
@@ -229,7 +229,7 @@ function dsp.battlefield.ExtendTimeLimit(battlefield, minutes, message, param, p
     if message then
         players = players or battlefield:getPlayers()
         for _, player in pairs(players) do
-            player:messageSpecial(message, param or minutes)
+            player:messageBasic(message, param or minutes)
```

Comment:
we might need 2 different versions as I am pretty sure some battlefields use a zone specific messageSpecial while others use the game-wide messageBasic, and IDs between the 2 are not the same.

If we wanna be fully retail accurate that is. I dunno if any currently use message Special, if none are I'd personally say screw those lets use the global one like you have this right now >.>

Reply:
Yeah this is currently only being used in Limbus, but I'm not sure where it would be used in the future so I just reverted it back to messageBasic and made a new function specific to Limbus. What other battlefields even have time extensions? Limbus uses a zone specific message "Your time in Limbus has been extended." followed by a Limbus specific time-remaining message.

Reply:
github seems to have taken away my ability to jump to the line we are talking on from the comment here, but I initially mistook this for the time remaining general message. Which..every bf has in some form, inc every bcnm isnm dyna limbo etc etc _everything._ I think they don't all use the same one normally (which is kinda annoying).